src/communication-security/grpc-code/samplebuff/sample.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option go_package = "github.com/pypalkar23/go-rpc-cis5209/sample";
```

Comment:
Perhaps, this name should be changed to something else like `pentest/grpc/samplebuff`?